core/src/main/java/org/apache/accumulo/core/data/impl/KeyExtent.java

```diff
@@ -383,32 +384,14 @@ public static Mutation getPrevRowUpdateMutation(KeyExtent ke) {
     return m;
   }
 
+  private static final Comparator<KeyExtent> COMPARATOR = Comparator
+      .comparing(KeyExtent::getTableId)
+      .thenComparing(KeyExtent::getEndRow, Comparator.nullsLast(Text::compareTo))
```

Comment:
This could use an explanation about why nulls are last, instead of first, which tends to be the norm.

Reply:
I made an attempt to comment this, but could not come up with anything I liked.  If you can supply a comment, I can add it.

Reply:
I can't provide a comment because I don't understand the reason for it. If you know the reason, I might be able to suggest a way to phrase it effectively.

Reply:
For endRow, null represents max.   For prevEndRow, null represents minimum.

Reply:
Oh, I think I understand now. How about something like:
```java
// The last tablet in a table has no end row, so null sorts last for end row; similarly, the first tablet has no previous end row, so null sorts first for previous end row
```